<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialogroomshow</class>
 <widget class="QDialog" name="Dialogroomshow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>437</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>437</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>640</width>
    <height>437</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>房间信息</string>
  </property>
  <property name="windowIcon">
   <iconset resource="hotel.qrc">
    <normaloff>:/new/prefix1/Images/Common/LOGO.png</normaloff>:/new/prefix1/Images/Common/LOGO.png</iconset>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>621</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/new/prefix1/Images/Common/roomshow.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>621</width>
     <height>351</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>房间信息</string>
    </attribute>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>房间号码</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser_roomnumber">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>10</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser_roomtype">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>60</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>70</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>房间类型</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>70</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>房间价格</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser_roomprice">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>60</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>120</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>房间状态</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser_roomsign">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>110</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>120</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>客户姓名</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser_clientname">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>110</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser_consumption">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>160</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>170</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>额外消费</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>170</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>已住天数</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser_day">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>160</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser_timein">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>210</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>220</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>入住时间</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>220</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>计划离店</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser_leavetime">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>210</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>270</y>
       <width>93</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>确定</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>额外消费</string>
    </attribute>
    <widget class="QTableWidget" name="tableWidget_pricetab">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>201</width>
       <height>301</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="tabKeyNavigation">
      <bool>true</bool>
     </property>
     <property name="alternatingRowColors">
      <bool>false</bool>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>84</number>
     </attribute>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <column>
      <property name="text">
       <string>名称</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>单价</string>
      </property>
     </column>
     <item row="0" column="0">
      <property name="text">
       <string>益达口香糖</string>
      </property>
     </item>
     <item row="0" column="1">
      <property name="text">
       <string>5</string>
      </property>
      <property name="font">
       <font>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="textAlignment">
       <set>AlignHCenter|AlignVCenter|AlignCenter</set>
      </property>
     </item>
     <item row="1" column="0">
      <property name="text">
       <string>雪花啤酒</string>
      </property>
     </item>
     <item row="1" column="1">
      <property name="text">
       <string>5</string>
      </property>
      <property name="textAlignment">
       <set>AlignHCenter|AlignVCenter|AlignCenter</set>
      </property>
     </item>
     <item row="2" column="0">
      <property name="text">
       <string>康师傅红茶</string>
      </property>
     </item>
     <item row="2" column="1">
      <property name="text">
       <string>3</string>
      </property>
      <property name="textAlignment">
       <set>AlignHCenter|AlignVCenter|AlignCenter</set>
      </property>
     </item>
     <item row="3" column="0">
      <property name="text">
       <string>康师傅绿茶</string>
      </property>
     </item>
     <item row="3" column="1">
      <property name="text">
       <string>3</string>
      </property>
      <property name="textAlignment">
       <set>AlignHCenter|AlignVCenter|AlignCenter</set>
      </property>
     </item>
     <item row="4" column="0">
      <property name="text">
       <string>可乐</string>
      </property>
     </item>
     <item row="4" column="1">
      <property name="text">
       <string>3</string>
      </property>
      <property name="textAlignment">
       <set>AlignHCenter|AlignVCenter|AlignCenter</set>
      </property>
     </item>
     <item row="5" column="0">
      <property name="text">
       <string>尖叫</string>
      </property>
     </item>
     <item row="5" column="1">
      <property name="text">
       <string>4</string>
      </property>
      <property name="textAlignment">
       <set>AlignHCenter|AlignVCenter|AlignCenter</set>
      </property>
     </item>
     <item row="6" column="0">
      <property name="text">
       <string>红牛</string>
      </property>
     </item>
     <item row="6" column="1">
      <property name="text">
       <string>5</string>
      </property>
      <property name="textAlignment">
       <set>AlignHCenter|AlignVCenter|AlignCenter</set>
      </property>
     </item>
     <item row="7" column="0">
      <property name="text">
       <string>水动乐</string>
      </property>
     </item>
     <item row="7" column="1">
      <property name="text">
       <string>4</string>
      </property>
      <property name="textAlignment">
       <set>AlignHCenter|AlignVCenter|AlignCenter</set>
      </property>
     </item>
     <item row="8" column="0">
      <property name="text">
       <string>大果粒</string>
      </property>
     </item>
     <item row="8" column="1">
      <property name="text">
       <string>7</string>
      </property>
      <property name="textAlignment">
       <set>AlignHCenter|AlignVCenter|AlignCenter</set>
      </property>
     </item>
     <item row="9" column="0">
      <property name="text">
       <string>芬达</string>
      </property>
     </item>
     <item row="9" column="1">
      <property name="text">
       <string>3</string>
      </property>
      <property name="textAlignment">
       <set>AlignHCenter|AlignVCenter|AlignCenter</set>
      </property>
     </item>
     <item row="10" column="0">
      <property name="text">
       <string>矿泉水</string>
      </property>
     </item>
     <item row="10" column="1">
      <property name="text">
       <string>2</string>
      </property>
      <property name="textAlignment">
       <set>AlignHCenter|AlignVCenter|AlignCenter</set>
      </property>
     </item>
     <item row="11" column="0">
      <property name="text">
       <string>雪碧</string>
      </property>
     </item>
     <item row="11" column="1">
      <property name="text">
       <string>3</string>
      </property>
      <property name="textAlignment">
       <set>AlignHCenter|AlignVCenter|AlignCenter</set>
      </property>
     </item>
     <item row="12" column="0">
      <property name="text">
       <string>王老吉</string>
      </property>
     </item>
     <item row="12" column="1">
      <property name="text">
       <string>4</string>
      </property>
      <property name="textAlignment">
       <set>AlignHCenter|AlignVCenter|AlignCenter</set>
      </property>
     </item>
     <item row="13" column="0">
      <property name="text">
       <string>果粒橙</string>
      </property>
     </item>
     <item row="13" column="1">
      <property name="text">
       <string>7</string>
      </property>
      <property name="textAlignment">
       <set>AlignHCenter|AlignVCenter|AlignCenter</set>
      </property>
     </item>
     <item row="14" column="0">
      <property name="text">
       <string>香飘飘奶茶</string>
      </property>
     </item>
     <item row="14" column="1">
      <property name="text">
       <string>6</string>
      </property>
      <property name="textAlignment">
       <set>AlignHCenter|AlignVCenter|AlignCenter</set>
      </property>
     </item>
     <item row="15" column="0">
      <property name="text">
       <string>软中华</string>
      </property>
     </item>
     <item row="15" column="1">
      <property name="text">
       <string>50</string>
      </property>
      <property name="textAlignment">
       <set>AlignHCenter|AlignVCenter|AlignCenter</set>
      </property>
     </item>
     <item row="16" column="0">
      <property name="text">
       <string>康师傅泡面</string>
      </property>
     </item>
     <item row="16" column="1">
      <property name="text">
       <string>6</string>
      </property>
      <property name="textAlignment">
       <set>AlignHCenter|AlignVCenter|AlignCenter</set>
      </property>
     </item>
     <item row="17" column="0">
      <property name="text">
       <string>安全套</string>
      </property>
     </item>
     <item row="17" column="1">
      <property name="text">
       <string>20</string>
      </property>
      <property name="textAlignment">
       <set>AlignHCenter|AlignVCenter|AlignCenter</set>
      </property>
     </item>
     <item row="18" column="0">
      <property name="text">
       <string>鸡腿</string>
      </property>
     </item>
     <item row="18" column="1">
      <property name="text">
       <string>20</string>
      </property>
      <property name="textAlignment">
       <set>AlignHCenter|AlignVCenter|AlignCenter</set>
      </property>
     </item>
     <item row="19" column="0">
      <property name="text">
       <string>鸡翅</string>
      </property>
     </item>
     <item row="19" column="1">
      <property name="text">
       <string>18</string>
      </property>
      <property name="textAlignment">
       <set>AlignHCenter|AlignVCenter|AlignCenter</set>
      </property>
     </item>
     <item row="20" column="0">
      <property name="text">
       <string>扑克</string>
      </property>
     </item>
     <item row="20" column="1">
      <property name="text">
       <string>5</string>
      </property>
      <property name="textAlignment">
       <set>AlignHCenter|AlignVCenter|AlignCenter</set>
      </property>
     </item>
     <item row="21" column="0">
      <property name="text">
       <string>早餐券</string>
      </property>
     </item>
     <item row="21" column="1">
      <property name="text">
       <string>15</string>
      </property>
      <property name="textAlignment">
       <set>AlignHCenter|AlignVCenter|AlignCenter</set>
      </property>
     </item>
     <item row="22" column="0">
      <property name="text">
       <string>午餐券</string>
      </property>
     </item>
     <item row="22" column="1">
      <property name="text">
       <string>30</string>
      </property>
      <property name="textAlignment">
       <set>AlignHCenter|AlignVCenter|AlignCenter</set>
      </property>
     </item>
     <item row="23" column="0">
      <property name="text">
       <string>晚餐劵</string>
      </property>
     </item>
     <item row="23" column="1">
      <property name="text">
       <string>25</string>
      </property>
      <property name="textAlignment">
       <set>AlignHCenter|AlignVCenter|AlignCenter</set>
      </property>
     </item>
     <item row="24" column="0">
      <property name="text">
       <string>花生</string>
      </property>
     </item>
     <item row="24" column="1">
      <property name="text">
       <string>4</string>
      </property>
      <property name="textAlignment">
       <set>AlignHCenter|AlignVCenter|AlignCenter</set>
      </property>
     </item>
     <item row="25" column="0">
      <property name="text">
       <string>薯片</string>
      </property>
     </item>
     <item row="25" column="1">
      <property name="text">
       <string>5</string>
      </property>
      <property name="textAlignment">
       <set>AlignHCenter|AlignVCenter|AlignCenter</set>
      </property>
     </item>
     <item row="26" column="0">
      <property name="text">
       <string>火腿肠</string>
      </property>
     </item>
     <item row="26" column="1">
      <property name="text">
       <string>3</string>
      </property>
      <property name="textAlignment">
       <set>AlignHCenter|AlignVCenter|AlignCenter</set>
      </property>
     </item>
     <item row="27" column="0">
      <property name="text">
       <string>开心果</string>
      </property>
     </item>
     <item row="27" column="1">
      <property name="text">
       <string>5</string>
      </property>
      <property name="textAlignment">
       <set>AlignHCenter|AlignVCenter|AlignCenter</set>
      </property>
     </item>
     <item row="28" column="0">
      <property name="text">
       <string>牛肉干</string>
      </property>
     </item>
     <item row="28" column="1">
      <property name="text">
       <string>8</string>
      </property>
      <property name="textAlignment">
       <set>AlignHCenter|AlignVCenter|AlignCenter</set>
      </property>
     </item>
    </widget>
    <widget class="QTableWidget" name="tableWidget_accounttab">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>10</y>
       <width>391</width>
       <height>251</height>
      </rect>
     </property>
     <property name="tabKeyNavigation">
      <bool>true</bool>
     </property>
     <property name="showDropIndicator" stdset="0">
      <bool>true</bool>
     </property>
     <property name="dragDropOverwriteMode">
      <bool>true</bool>
     </property>
     <property name="alternatingRowColors">
      <bool>false</bool>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>94</number>
     </attribute>
     <attribute name="verticalHeaderMinimumSectionSize">
      <number>0</number>
     </attribute>
     <column>
      <property name="text">
       <string>名称</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>单价</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>数量</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>金额</string>
      </property>
     </column>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>290</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>合计:</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser_total">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>280</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_yesforbuy">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>280</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>确定</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="hotel.qrc"/>
 </resources>
 <connections/>
</ui>
